src/contracts/core/DelegationManager.sol

```diff
@@ -68,18 +68,21 @@ contract DelegationManager is Initializable, OwnableUpgradeable, Pausable, Deleg
 
     /**
      * @dev Initializes the addresses of the initial owner, pauser registry, and paused status.
-     * withdrawalDelayBlocks is set only once here
+     * strategyWithdrawalDelayBlocks is set only once here
```

Comment:
small nit: seems like the key thing set once is actually the `minWithdrawalDelayBlocks` variable

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/392/commits/5d8e5ff14d1eee680465fa4a712eef41033cb9a6